addons/point_of_sale/models/product_combo.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProductCombo(models.Model):
+    _name = "product.combo"
```

Comment:
```suggestion
    _name = "pos.product.combo"
```
do the same prefixing to the other models.